### PR TITLE
Create Sticky footer

### PR DESCRIPTION
**Pull request by N6REJ**
Pull Request for Issue # .

### Summary of Changes
Allows for sticky footer
removes trailing space when sticky is not used.

### Testing Instructions
observe back-to-top only shows if your at the very bottom of the page
apply pr.
in template styles turn on "sticky footer"
run npm ci
observe back-to-top stays @ bottom of view port

### Expected result
image

image

### Actual result
image
image

### CODE REVIEW

**Comment by richard67** [comment]
Now this PR shows no changes at all.

**Comment by N6REJ** [comment]
> Now this PR shows no changes at all.

I fell asleep 👎

File: templates/cassiopeia/templateDetails.xml [hunk 1]
```diff
@@ -126,6 +126,18 @@
 					<option value="1">JYES</option>
 				</field>
 
+				<field
+						name="stickyFooter"
+						type="radio"
+						label="TPL_CASSIOPEIA_STICKY_FOOTER_LABEL"
+						layout="joomla.form.field.radio.switcher"
+						default="0"
+						filter="integer"
```
Comment (richard67) [inline templates/cassiopeia/templateDetails.xml]:
```suggestion
					name="stickyFooter"
					type="radio"
					label="TPL_CASSIOPEIA_STICKY_FOOTER_LABEL"
					layout="joomla.form.field.radio.switcher"
					default="0"
					filter="integer"
```

**Comment by hans2103** [comment]
@N6REJ please no sticky footer.

If back-to-top should be visible at all time I would suggest to change the css to make back-to-top sticky. Not the entire footer. 
I would recommend to set back-to-top sticky and add some js to let it appear after x amount of scroll.

**Comment by richard67** [comment]
Agree with the above statement of Hans.

**Comment by brianteeman** [comment]
or better yet don't use any javascript at all
https://codepen.io/rolandtoth/details/eMamVK

**Comment by N6REJ** [comment]
> @N6REJ please no sticky footer.
> 
> If back-to-top should be visible at all time I would suggest to change the css to make back-to-top sticky. Not the entire footer.
> I would recommend to set back-to-top sticky and add some js to let it appear after x amount of scroll.

what "title" would you use for the switch?

**Comment by N6REJ** [comment]
going to close this and focus on just the BTT

**Pull request closed without merging**